grails-app/migrations/views/cycle-count-session.sql

```diff
@@ -8,10 +8,10 @@ SELECT
     product_availability.location_id                                                 as facility_id,
 
     -- Cycle count request ID
-    cycle_count_request_summary.cycle_count_request_id                               as cycle_count_request_id,
+    MAX(cycle_count_request_summary.cycle_count_request_id)                               as cycle_count_request_id,
```

Comment:
I'm hoping this resolves the issue that Evan brought up in Slack (https://openboxes.slack.com/archives/C06JCHWR807/p1741627530593889).